AIDevGallery/Samples/Open Source Models/Language Models/CustomSystemPrompt.xaml.cs

```diff
@@ -43,7 +43,7 @@ public CustomSystemPrompt()
     protected override async Task LoadModelAsync(SampleNavigationParameters sampleParams)
     {
         model = await sampleParams.GetIChatClientAsync();
-        InputTextBox.MaxLength = chatOptions.MaxOutputTokens ?? 0;
+        InputTextBox.MaxLength = GenAIModel.DefaultMaxLength;
```

Comment:
All other language samples use GenAIModel.DefaultMaxLength, and ChatOptions.MaxOutputTokens are the maximum output tokens, not the maximum input length. 
It would make sense (eventually) to add a new slider, but I don't think it's needed

Reply:
The other samples use the constants since they don't have an instance of `chatOptions`, but I see what you mean.
Lets open a separate issue and deal with this on a separate PR. This is important, but just not part of the same crash/issue.